lib/robots/dor_repo/ocr/stage_files.rb

```diff
@@ -9,9 +9,18 @@ def initialize
           super('ocrWF', 'stage-files')
         end
 
```

Comment:
This context helped ensure the workspace was setup properly for each test. I was using TMP_DIR which worked locally but failed in CI for some reason that I couldn't quite figure out.